submit_and_compare/submit_and_compare.py

```diff
@@ -131,22 +129,26 @@ class SubmitAndCompareXBlock(XBlock):
 
     has_score = True
 
-    '''
+    """
     Main functions
-    '''
+    """
     def student_view(self, context=None):
-        '''
+        # pylint: disable=unused-argument
+        """
         The primary view of the XBlock, shown to students
         when viewing courses.
-        '''
+        """
         problem_progress = self._get_problem_progress()
         used_attempts_feedback = self._get_used_attempts_feedback()
         submit_class = self._get_submit_class()
-        prompt = self._get_body(self.question_string)
-        explanation = self._get_explanation(self.question_string)
-
+        prompt = SubmitAndCompareXBlock._get_body(self.question_string)
```

Comment:
1. Let's make this a function in the module, instead of a method in the class (define it at the top level),
since it doesn't need access to anything within the class/object.
2. In Python, a leading underscore implies a private value. Since we need to call it from outside of this module (in the tests), it should be "public" and have the leading `_` removed.